src/graph.rs

```diff
@@ -1550,13 +1552,33 @@ fn resolve(
   specifier_text: &str,
   referrer_range: Range,
   maybe_resolver: Option<&dyn Resolver>,
+  maybe_npm_resolver: Option<&dyn NpmResolver>,
 ) -> Resolution {
   let response = if let Some(resolver) = maybe_resolver {
     resolver.resolve(specifier_text, &referrer_range.specifier)
   } else {
     resolve_import(specifier_text, &referrer_range.specifier)
       .map_err(|err| err.into())
   };
+  use ResolveError::*;
+  use SpecifierError::*;
+  if let Err(Specifier(ImportPrefixMissing(_, _))) = response.as_ref() {
+    if let Some(npm_resolver) = maybe_npm_resolver {
+      let maybe_specifier =
+        ModuleSpecifier::parse(&format!("node:{}", specifier_text)).ok();
+      let maybe_mod_name = maybe_specifier.as_ref().and_then(|s| {
+        npm_resolver.resolve_builtin_node_module(s).ok().flatten()
+      });
+      if maybe_mod_name.is_some() {
+        npm_resolver.on_resolve_bare_builtin_node_module(specifier_text);
+        return Resolution::from_resolve_result(
+          Ok(maybe_specifier.unwrap()),
```

Comment:
FWIW, the `.unwrap()` could be removed by using `.filter` on the result of `ModuleSpecifier::parse(&format!("node:{}", specifier_text)).ok()` when `npm_resolver.resolve_builtin_node_module(s).ok().flatten().is_some()`

Reply:
> FWIW, the .unwrap() could be removed by using .filter on the result of ModuleSpecifier::parse(&format!("node:{}", specifier_text)).ok()

I didn't get what this exactly means.., but removed the usage of `.unwrap()` in a different way. https://github.com/denoland/deno_graph/pull/295/commits/4ebda530c0c60584707c9a8aee3f74d14f0457f4